### PR TITLE
chore(suite-native): remove add account feature flag

### PR DESCRIPTION
**Pull request by vytick**
Remove feature flag for Add account since it is not needed anymore

### CODE REVIEW

File: suite-native/accounts/src/components/AddAccountsButton.tsx [hunk 1]
```diff
@@ -38,7 +36,7 @@ export const AddAccountButton = ({ flowType }: AddAccountButtonProps) => {
     };
 
     const navigateToAddCoinAccount = () => {
-        if (isAddCoinAccountEnabled) {
+        if (isDeviceConnectEnabled) {
```
Comment (PeKne) [inline suite-native/accounts/src/components/AddAccountsButton.tsx]: Is not this condition redundant now? If `isDeviceConnectEnabled` === `false` user should not be able to execute this function anyway.

Comment (vytick) [reply suite-native/accounts/src/components/AddAccountsButton.tsx]: that is a failsafe :) but yes, he should not

Comment (PeKne) [reply suite-native/accounts/src/components/AddAccountsButton.tsx]: I would vote for removing it then to make the code as simple as possible. But I will leave this up to you. Approving :)

**Review by PeKne — APPROVED** [review]

File: suite-native/accounts/src/components/AddAccountsButton.tsx (hunk 1 above)
Comment (vytick) [reply suite-native/accounts/src/components/AddAccountsButton.tsx]: updated

**Pull request merged**